R/react.R

```diff
@@ -157,22 +157,89 @@ JS <- function(...) { # nolint
 #' @export
 triggerEvent <- function(inputId) {
   ReactData(
-    type = "input", id = inputId, argIdx = NULL
+    type = "event", id = inputId
   )
 }
 
-#' Set input
+#' @title Set input
 #'
-#' Creates a handler which can be used for `onChange` and similar props of 'React' components
-#' to set the value of a 'Shiny' input to one of the arguments passed to the handler.
+#' @description Creates a handler which can be used for `onChange` and similar
```

Comment:
I'd suggest to drop `@title` and `@description`:
1. Consistency: that's how the rest of our docs are written.
2. The first paragraph will become the `@description` and the remaining ones (starting with "The argument...") will become `@details` which makes sense in the rendered docs.

Reply:
Done!